### PR TITLE
release: v1.32.0

### PR DESCRIPTION
**Pull request by nkubala**

### CODE REVIEW

File: examples/custom-tests/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ test:
           - <paths glob>
 ```
 
-As tests take time, you might prefer to configure tests using [profiles](https://skaffold.dev/docs/environment/profiles/) so that they can be automatically enabled or disabled, e.g.
+As tests take time, you might prefer to configure tests using [profiles](https://skaffold.dev/docs/https://skaffold.dev/docs/environment/profiles/) so that they can be automatically enabled or disabled, e.g.
```
Comment (briandealwis) [inline examples/custom-tests/README.md:20]: This change doesn't look right

**Review by briandealwis — APPROVED** [review]
1 nit to fix

File: examples/custom-tests/README.md (hunk 1 above)
Comment (nkubala) [reply examples/custom-tests/README.md:20]: no it sure doesn't, this was autogenerated by our release script...

Comment (nkubala) [reply examples/custom-tests/README.md]: this slipped through in https://github.com/GoogleContainerTools/skaffold/pull/5451, but i'm really not sure how it's just getting synced now. fixed anyway

**Pull request merged**